ui/plugin-system/src/model/plugins.ts

```diff
@@ -71,4 +71,5 @@ export type PluginImplementation<Type extends PluginType> = SupportedPlugins[Typ
 /**
  * Default plugin kinds by plugin type.
  */
-export type DefaultPluginKinds = Partial<Record<PluginType, string>>;
+type PluginKinds = Partial<Record<PluginType, string>>;
+export type DefaultPluginKinds = Required<Pick<PluginKinds, 'TimeSeriesQuery'>> & Omit<PluginKinds, 'TimeSeriesQuery'>;
```

Comment:
If anyone knows a more elegant way to write this, let me know 😄

Reply:
I'm trying to remember why I even wrote it the way it was previously. Probably the result of some fighting with TypeScript. We only use the defaults for `TimeSeriesQuery` and `Panel`, right? If so, I wonder if this can be simplified to something like this. That'd be a lot simpler.

```
export type DefaultPluginKinds = {
  TimeSeriesQuery: string;
  Panel?: string;
};
```